recipes/MultiWOZ/response_generation/train_with_gpt.py

```diff
@@ -426,9 +425,7 @@ def input_and_token_type_pipeline(
     )
 
     # Load tokenizer and add special tokens
-    tokenizer = GPT2Tokenizer.from_pretrained(

```

Comment:
@poonehmousavi : I removed these lines because the tokenizer will be automatically loaded by the GPT class.